en/docs/Administer/ProductSecurity/General/SAML2SSO/configuring-external-idp-through-identity-server-for-sso.md

```diff
@@ -207,7 +207,7 @@ Similarly, provide the configurations to register the API Store as the SSO serv
 -   Assertion Consumer URL: `                           https://localhost:9443/store/jagg/jaggery_acs.jag                         ` Change the IP and port accordingly. This is the URL for the acs page in your running store app.
 -   Select the following options:
     -   **Enable Response Signing**
-    -   **Enable** **Single Logout**
+    -   **EnableSingle Logout**
```

Comment:
space is missing

Reply:
done

Reply:
Fixed in Commit on Sep 9, 2019 - https://github.com/wso2/docs-apim/pull/19/commits/d2c3ff8450dd3cdcef80a86027b482e4ec4f5410